pkg/shares/non_interactive_defaults_test.go

```diff
@@ -236,7 +230,7 @@ func TestNextMultipleOfBlobMinSquareSize(t *testing.T) {
 			blobLen:       2,
 			squareSize:    8,
 			fits:          true,
-			expectedIndex: 12,
+			expectedIndex: 11,
 		},
 		{
 			// inspired by the diagram at https://github.com/celestiaorg/celestia-app/blob/1b80b94a62c8c292f569e2fc576e26299985681a/docs/architecture/adr-009-non-interactive-default-rules-for-reduced-padding.md?plain=1#L30
```

Comment:
proposal to delete this comment b/c the diagram from ADR9 and the test case now diverge.